prow/plugins/assign/assign.go

```diff
@@ -188,7 +188,7 @@ type handler struct {
 func newAssignHandler(e github.GenericCommentEvent, gc githubClient, log *logrus.Entry) *handler {
 	org := e.Repo.Owner.Login
 	addFailureResponse := func(mu github.MissingUsers) string {
-		return fmt.Sprintf("GitHub didn't allow me to assign the following users: %s.\n\nNote that only [%s members](https://github.com/orgs/%s/people) and repo collaborators can be assigned.", strings.Join(mu.Users, ", "), org, org)
+		return fmt.Sprintf("GitHub didn't allow me to assign the following users: %s.\n\nNote that only [%s members](https://github.com/orgs/%s/people) and repo collaborators can be assigned. \nFor more information please see [here](https://git.k8s.io/community/contributors/guide/#issue-assignment-in-github)", strings.Join(mu.Users, ", "), org, org)
```

Comment:
tiny nit(s)/thought..
WDYT: `s/here/the contributor guide/`? And maybe add a period after?

Reply:
:paintbrush: :art: :bike: :derelict_house:

Reply:
Hmmm, I see you on "the contributor guide" being more visible.
Other prow users would have to change the link anyway, or is there a bit of automation on configuration that I'm unaware of? Until and unless we make those bot strings into variables, it's pretty k8s specific...

Reply:
yeah I think the better answer is to make this more configurable in the future..
cc @stevekuznetsov @sebastienvas

Reply:
How would prow configuration work though? Is there a design doc/ existing partial implementation? Something like a vars file that bot functions refer to?
In any case, to stay in scope here, I'll sub in "the contributor guide" for now but would be excited to see follow up on configuration. :)

Reply:
the one example so far is: 
https://github.com/kubernetes/test-infra/blob/c51ad9418585943248a013cb09302e459dfbf785/prow/config.yaml#L3

probably we'd add this to the config in `prow/plugins.yaml` for this config as a go template operating over an info struct.

that's super out of the scope for this PR though, and probably needs a much broader fix.
The upshot to this would be instant update of the message formatting on PR merge if it didn't need any changes to the struct.